variables.tf

```diff
@@ -161,10 +161,10 @@ variable "use_fips_ssl" {
   default     = "false"
 }
 
-variable "wlm_json_configuration" {
+variable "wlm_configuration" {
```

Comment:
Let's keep the existing variable `wlm_json_configuration` and add new variable `wlm_configuration` which will be used if not empty inside the value of `aws_redshift_parameter_group`.

Reply:
done